doc/modules/preprocessing.rst

```diff
@@ -299,8 +302,42 @@ This can be confirmed on a independent testing set with similar remarks::
   ... # doctest: +ELLIPSIS +SKIP
   array([ 0.01...,  0.25...,  0.46...,  0.60... ,  0.94...])
 
-It is also possible to map the transformed data to a normal distribution by
-setting ``output_distribution='normal'``::
+Mapping to a Gaussian distribution
+----------------------------------
+
+In many modeling scenarios, normality of the features in a dataset is desirable.
+Power transforms are a family of parameteric transformations that aim to
+map data from any distribution to as close to a Gaussian distribution as
+possible.
+
+:class:`BoxCoxTransformer` provides one such power transformation.
+The Box-Cox transform is given by:
+
+.. math::
+    y_i^{(\lambda)} =
+    \begin{cases}
+    \dfrac{y_i^\lambda - 1}{\lambda} & \text{if } \lambda \neq 0, \\[8pt]
+    \ln{(y_i)} & \text{if } \lambda = 0,
+    \end{cases}
+
+The transformation is parameterized by :math:`\lambda`, which is determined
+through maximum likelihood estimation. Here is an example of using Box-Cox to
+map samples drawn from a lognormal distribution to a normal distribution::
+
+  >>> boxcox = preprocessing.BoxCoxTransformer()
```

Comment:
embed a picture from an example? I feel for this a picture can be very helpful.

Reply:
Done - added `examples/preprocessing/plot_power_transformer.py`.